course5/homework1.1/homework1.1.cc

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+#include <set>
+
+int main(int argc, char** argv)
+{
+	std::set<std::string> words;
```

Comment:
Also consider std::multiset, as you can get the same word multiple times, this way the duplicates get lost.